packages/tabs/partials/TabTriggers/style.css

```diff
@@ -72,6 +72,7 @@
           max-width: none;
           padding-left: 48px;
           padding-right: 60px;
+          width: auto;
```

Comment:
Secret sauce for the arrow thing. `.g-grid-container` was applying `width: 100%;` to the inner container of the tab triggers, so the element dimensions weren't truly correct and the arrow logic was broken as a result.